modules/local/format_taxonomy.nf

```diff
@@ -31,7 +31,6 @@ process FORMAT_TAXONOMY {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        bash: \$(bash --version | sed -n 1p | sed 's/GNU bash, version //g')
-    END_VERSIONS
+        bash: \$(bash --version | sed -n 1p | grep -Eo 'version [[:alnum:].]+' | sed 's/version //')
```

Comment:
```suggestion
        bash: \$(bash --version | sed -n 1p | grep -Eo 'version [[:alnum:].]+' | sed 's/version //')
        END_VERSIONS
```
Doesn't one need this? (I see you removed it in several places.)

Reply:
Yes that is really weird: The `END_VERSIONS` was ending up in the `versions.yml` and the pipeline failed. So I removed it and the tests passed. I am not sure why. If there is a better solution, that would be great indeed, any idea?

Reply:
That sounds like it's oddly placed in the generated script. It should be to right to the left without whitespace, I think.

Reply:
I see that my suggestion will actually place whitespaces before `END_VERSIONS`, so don't accept that!

Reply:
Thanks, I'll have a look

Reply:
Indeed, there was weird indentation after variables which hold scripts from bin. We'll see if thats resolved now.